build.zig.zon

```diff
@@ -3,8 +3,8 @@
     .version = "0.1.0",
     .dependencies = .{
         .zigx = .{
-            .url = "https://github.com/marler8997/zigx/archive/454432fd1ad0ae0f1a949b6f1828cffcce3bb4ce.tar.gz",
-            .hash = "1220a6ba0703e5baab2c5eb18931c66e4a73b59a298c24ed03f588c80e7c3f82a2ea",
+            .url = "https://github.com/MadLittleMods/zigx/archive/82fb1b754261dd683f16f8cffeabf41c3fc312a8.tar.gz",
```

Comment:
TODO: This commit needs to be updated to something that includes both the changes from https://github.com/marler8997/zigx/pull/8 and https://github.com/marler8997/zigx/pull/9